File: Application/Services/Operations/ProductServices/Dtos/Fillers/GetTogetherDto.cs
using System;
using System.Collections.Generic;
using Application.Services.Operations.Main.Companies.Dtos;

namespace Application.Services.Operations.ProductServices.Dtos.Fill
{
    public class GetTogetherDto
    {
        public List<Equipament_FillDto> Equipaments_Fill { get; set; }
        public List<Manufacturer_FillDto> Manufacturers_Fill { get; set; }
        public List<Segment_FillDto> Segments_Fill { get; set; }
    }
}

File: Views/src/components/product/dtos/get-together-dto.ts
import { EquipamentFillDto } from "./equipament-fill-dto";
import { ManufacturerFillDto } from "./manufacturer-fill-dto";
import { SegmentFillDto } from "./segment-fill-dto";

export class GetTogetherDto {
  equipaments_Fill: EquipamentFillDto[];
  manufacturers_Fill: ManufacturerFillDto[];
  segments_Fill: SegmentFillDto[];
}
